src/vmshepherd_runtime_postgres_driver/__init__.py

```diff
@@ -63,18 +88,25 @@ def reconfigure(self, config):
                  'failed_checks': vms_states['failed_checks']
                }
 
-    async def _acquire_lock(self, name):
+    async def _acquire_lock(self, preset_name):
```

Comment:
Parameter's name is a part of public API. Is this necessary?

Reply:
It's for unifacation of the arguments name, in vmshepherd we not use kwargs when we call this method, so its backword compability